napari_animation/interpolation.py

```diff
@@ -9,7 +9,7 @@
 
 def default(a, b, fraction):
     """Default interpolation for the corresponding type;
-    linear interpolation for numeric, step interpolation otherwise.
+    linear interpolation for numeric, instantaneous transition otherwise.
```

Comment:
```suggestion
    linear interpolation for numeric, step change at start of transition otherwise.
```

slightly more precise, feel free to modify if you feel strongly!